File: A/Rec/sort.py
def selectionSort(arr, n):
    for i in range(n - 1):
        min = i
        for j in range(i + 1, n):
            if arr[j] < arr[min]:
                min = j
        
        arr[i], arr[min] = arr[min], arr[i]


def bubbleSort(arr, n):
    for i in range(n-1, 0, -1):
        for j in range(0, i):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]


if __name__ == "__main__":
    n = int(input("Enter no of elements: "))
    arr = []
    for i in range(n):
        arr.append(int(input("Enter the element: ")))

    print(arr)

    bubbleSort(arr, n)
    print(arr)

    

File: A/sort.cpp
#include <iostream>
#include <vector>
using namespace std;

void selectionSort(int arr[], int n) {
    for(int i = 0; i <= n - 2; i++) {
        int min = i;
        for (int j = i + 1; j < n; j++) {
            if (arr[j] < arr[min]) min = j; 
        }

        int temp = arr[min];
        arr[min] = arr[i];
        arr[i] = temp;
    }
}

int main() {
    cout<<"Enter no of integers: " << endl;
    int n;
    cin >> n;
    vector<int> arr(n);
    cout<<"Enter elements of array: " << endl;
    for(int i = 0; i < n; i++) cin >> arr[i];
    selectionSort(arr.data(), n);
    for(int i = 0; i < n; i++) cout << arr[i] << " ";
    cout << endl;
    return 0;
}
